ynr/apps/people/management/commands/people_clean_person_names.py

```diff
@@ -0,0 +1,35 @@
+from candidatebot.helpers import CandidateBot
+from django.core.management.base import BaseCommand
+from people.models import Person
+from popolo.models import OtherName
+
+
+class Command(BaseCommand):
+    help = "Replaces chars in person names that we don't want to be there."
+
+    def handle(self, *args, **options):
+        chars_to_replace = {"\xa0": " ", "\u200e": ""}
```

Comment:
This feels like a bit of a reactive whack-a-mole-ey solution. Its fine, but at some point, someone is going to rock up with a `\u2007` in the name field and we won't catch it.

Its a bit more effort to think about, but I feel like we could define some slightly more general rules that would catch this problem and also similar future problems using [unicode chracter classes](https://www.unicode.org/reports/tr44/#GC_Values_Table). Here's an example for the Space_Separator category (Zs):

```py
import sys
import unicodedata

def normalise_whitespace(text):
    whitespace_map = {
        ord(c): " "
        for c in map(chr, range(sys.maxunicode))
        if unicodedata.category(c) == "Zs"
    }
    return text.translate(whitespace_map)

text = "Some text\u202fwith\xa0non-standard\u2007whitespace."
print(normalise_whitespace(text))
```